### DISCUSSION TITLE
No puedo clonar ninguna repo

### DISCUSSION THREAD

**Discussion by JeanFranco99** [INGCOM-UNRN-P1/cursada-2023#6] [Preguntas y respuestas] [answered]
Buenas! Estoy teniendo problemas para clonar la repo del tp0 tp1, ya sea usando la consola de comando git, github destkop y gitkraken.
Con la consola me sale esto. Intente hacer lo de los links pero sigue sin funcionar
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/a849e253-6122-4b8f-aa10-ccf4ddfe1645)
Con github desktop me sale esto una y otra vez 
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/213c1a39-8731-411d-a17d-8a3d4ed4d441)
Y con gitkraken es algo parecido
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/83616156/ada23c83-0154-4cb4-8250-80838545c610)
Con el de SSH me pasa lo mismo

**Comment by martinvilu** [comment]
El problema está en el uso de Gitkraken, que solicita permisos adicionales para poder completar la acción de clonado.

De la primera captura, veo que hay una solicitud de usuario y contraseña; esta faltando configurar la llave ssh para completar la operación.

**Reply by martinvilu** [reply]
Fijate que te dice si usas la dirección clone ssh: `git@github.com:INGCOM-UNRN-P1/tp1-2023-JeanFranco99.git` para el repositorio del TP1.

**Reply by JeanFranco99** [reply, marked as answer]
Ahí tuve que configurar la clave SSH en la computadora y pude clonar la repo sin problemas🫡
